src/status_im/subs/communities.cljs

```diff
@@ -67,6 +61,41 @@
           (sort-by #(get names (get % 0)))
           (sort-by #(visibility-status-utils/visibility-status-order (get % 0)))))))
 
+(re-frame/reg-sub
+ :communities/sorted-community-members-section-list
```

Comment:
This sub could really get some refactors. This is not status-go where we functions have hundreds of lines of code and that's okay for everybody ;)

Jokes aside, more importantly, I don't see unit tests for it and it should have plenty given the amount of logic. This is in our guidelines too.

Reply:
Added a test, Will try to refactor it.